src/database/index.tsx

```diff
@@ -540,6 +540,22 @@ export const removeScenarioUser = async (
   }
 };
 
+const batchSetFacilityModelVersions = async (
+  modelVersions: ModelInputs[],
+  facilityDocRef: firebase.firestore.DocumentReference,
+  batch: firebase.firestore.WriteBatch,
+) => {
+  modelVersions.forEach((modelVersion: ModelInputs) => {
+    const modelVersionDoc = facilityDocRef
```

Comment:
You should be able to retrieve this modelVersionDoc outside of the `forEach` loop

Reply:
Nevermind, we're creating a new `doc` each time